wagtail_wordpress_import/importers/wordpress.py

```diff
@@ -335,13 +335,19 @@ def get_yoast_description_value(self):
         from the XML import file <item><description>...</description></item>
         """
         meta_value = ""
-
-        if yoast_plugin_config()["xml_item_key"] in self.node.keys():
-            for item in self.node.get(yoast_plugin_config()["xml_item_key"]):
-                meta_key_values = list(item.values())
-
-                if meta_key_values[0] == yoast_plugin_config()["description_key_value"]:
-                    meta_value = meta_key_values[1] or ""
+        node_keys = self.node.keys()
+        xml_item_key = yoast_plugin_config()["xml_item_key"]
+
+        if xml_item_key in node_keys:
+            for item in self.node.get(xml_item_key):
+                if type(item) == dict:
```

Comment:
**Suggestion**: use isinstance, as it covers inherited subtypes too

```suggestion
                if isinstance(item, dict):
```

Reply:
**Question** I'm not sure how to write a test that would fail without this fix?

but I have added a new test to check a single wp:postmeta runs successfully.

Reply:
Sorry, Nick, I only just saw this comment. Are you talking about the `isinstance` fix or the single vs. multiple `<wp:postmeta>` tags fix? The former doesn't need a unit test, the latter I can look at today.

Reply:
I was looking at it last night, the issue was when a None type is returned. I think I have a better idea for that function, shall I do that work before you review again? Also the next ticket codebase #73 for dealing with the import hooks might mean altering the process around yoast and other wp:postmeta elements.

Reply:
Second thoughts if poss and this fix is acceptable it might be better to stay with this for now.